scss/utilities/_text.scss

```
@@ -62,6 +62,8 @@
 
 .text-decoration-none { text-decoration: none !important; }
 
+.text-break    { word-break: break-word !important; }
```

One space should do the job.
```suggestion
.text-break { word-break: break-word !important; }
```